index.html

```diff
@@ -48,5 +48,116 @@ <h2>I am a Software Developer</h2>
       </div>
     </section>
 
+    <section class="work-section">
+      <div class="title">
+  <h1>My Recent Works</h1>
+      </div>
+    </section>
+
+ <section class="works">
+   <div class="my-works">
+     <div class="work-pic-1">
+       <img src="./img/work-img/work-1.png" alt="My work" />
+     </div>
+     <div class="work-content">
+       <h2>Multi-Post Stories</h2>
+       <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+       <div class="programming-languages">
+         <p>html</p>
+         <p>Bootstrap</p>
+         <p>Ruby</p>
+       </div>
+       <a href="#">See Project</a>
+     </div>
+   </div>
+   <div class="work-pic-2">
+     <h2>Profesional Art<br> Printing Data</h2>
+     <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+     <div class="programming-languages">
+       <p>html</p>
+       <p>Bootstrap</p>
+       <p>Ruby</p>
+     </div>
+     <a href="#">See Project</a>
+   </div>
+   <div class="work-pic-2">
+     <h2>Profesional Art<br> Printing Data</h2>
+     <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+     <div class="programming-languages">
+       <p>html</p>
+       <p>Bootstrap</p>
+       <p>Ruby</p>
+     </div>
+     <a href="#">See Project</a>
+   </div>
+   <div class="work-pic-2">
+     <h2>Profesional Art<br> Printing Data</h2>
+     <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+     <div class="programming-languages">
+       <p>html</p>
+       <p>Bootstrap</p>
+       <p>Ruby</p>
+     </div>
+     <a href="#">See Project</a>
+   </div>
+   <div class="work-pic-2">
+     <h2>Profesional Art<br> Printing Data</h2>
+     <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+     <div class="programming-languages">
+       <p>html</p>
+       <p>Bootstrap</p>
+       <p>Ruby</p>
+     </div>
+     <a href="#">See Project</a>
+   </div>
+   <div class="work-pic-2">
+     <h2>Profesional Art<br> Printing Data</h2>
+     <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+     <div class="programming-languages">
+       <p>html</p>
+       <p>Bootstrap</p>
+       <p>Ruby</p>
+     </div>
+     <a href="#">See Project</a>
+   </div>
+   </section>
+
+   <section class="About-me">
+     <h2>About me</h2>
+     <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+    <a href="#">Get My Resume</a>

```

Comment:
- [ ] I would have made this optional but while you are at it fixing the other changes, kindly change the font here and also make the "Get my resume" button a little bigger as shown in the Figma template.

<html>
<body>
<!--StartFragment--><p dir="auto" style="box-sizing: border-box; margin-top: 0px !important; margin-bottom: 16px; color: rgb(201, 209, 217); font-family: -apple-system, BlinkMacSystemFont, &quot;Segoe UI&quot;, Helvetica, Arial, sans-serif, &quot;Apple Color Emoji&quot;, &quot;Segoe UI Emoji&quot;; font-size: 14px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(13, 17, 23); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;"></p>

Your design | Figma design
-- | --
![image](https://user-images.githubusercontent.com/29071984/170837761-fd147a04-fed9-4dca-aa41-a2c9eb5e16f1.png)  |  ![image](https://user-images.githubusercontent.com/29071984/170837734-625adadc-c290-4c19-844d-f221f950fec6.png)


<!--EndFragment-->
</body>
</html>